update.sh

```diff
@@ -60,6 +60,8 @@ update_version()
 	case "$version" in
 		10.3 | 10.4)
 			sed -i -e '/--old-mode/d' "$version/docker-entrypoint.sh"
+			sed -i -e 's/REPLICATION REPLICA/REPLICATION SLAVE/' "$version/docker-entrypoint.sh"
```

Comment:
good. Could combine to a single `sed` command as all are changing the same file.

Reply:
Done 👍🏽